src/DependencyInjection/Configuration.php

```diff
@@ -671,10 +670,10 @@ private function getOrmEntityManagersNode(): ArrayNodeDefinition
                         ->prototype('scalar')->end()
                     ->end()
                     ->booleanNode('report_fields_where_declared')
-                        ->defaultValue(! class_exists(AnnotationDriver::class))
+                        ->defaultValue(true)
                         ->info('Set to "true" to opt-in to the new mapping driver mode that was added in Doctrine ORM 2.16 and will be mandatory in ORM 3.0. See https://github.com/doctrine/orm/pull/10455.')
                         ->validate()
-                            ->ifTrue(static fn (bool $v): bool => ! class_exists(AnnotationDriver::class) && ! $v)
+                            ->ifTrue(static fn (bool $v): bool => ! $v)
```

Comment:
I cannot use `ifFalse()` here, it's not always available, and anyway, this configuration is going to be removed in a subsequent PR.